<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClkWindowSettings</class>
 <widget class="QDialog" name="ClkWindowSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Clock Window Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="appSettings">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="customScaleCBox">
        <property name="text">
         <string>Scale font size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="customFontCBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Use custom font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFontComboBox" name="chooseFontBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="chooseScaleBox">
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="minimum">
         <number>80</number>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rclockSettings">
     <property name="title">
      <string>Roomclock</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1">
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="sechandmvmtCaption">
        <property name="text">
         <string>Second hand movement:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="showSecHand">
        <property name="text">
         <string>Show second hand</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="sharpRClock">
        <property name="text">
         <string>&amp;Sharp</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="smoothRClock">
        <property name="text">
         <string>S&amp;mooth</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="swissRClock">
        <property name="text">
         <string>Swiss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="lockSettingsBttn">
       <property name="text">
        <string>Lock</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/breeze-icons/object-locked-16.svg</normaloff>:/breeze-icons/object-locked-16.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeDialogBttn">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/breeze-icons/dialog-ok-apply-16.svg</normaloff>:/breeze-icons/dialog-ok-apply-16.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
